R/dependencies.R

```diff
@@ -579,13 +583,12 @@ renv_dependencies_discover_description_impl <- function(dcf, field, path, type)
     return(list())
 
   # create dependency list
-  dev <- field == "Suggests" && type != "package"
   renv_dependencies_list(
     path,
     extract_chr(matches, 2L),
     extract_chr(matches, 3L),
     extract_chr(matches, 4L),
-    dev
+    dev = field == "Suggests"
```

Comment:
FWIW I think the reason I had this was to avoid issues with DESCRIPTION files located in sub-directories within a project, but that imply separate considerations (e.g. should we really be parsing those for dependencies at all from the top-level?)